code/drasil-example/README.md

```diff
@@ -7,33 +7,51 @@ This folder contains our working examples (implementations in Drasil of the prev
 
 **dblpend**
   - Contains the current state of the Double Pendulum case study implemented in Drasil
+  - This is a recreation of the original, manually-created version 
```

Comment:
Through all of the examples you should say [recreation vs re-creation](https://www.google.com/search?q=recreation+vs+re-creation&sca_esv=596015288&rlz=1C5CHFA_enCA763CA763&sxsrf=ACQVn096xoPk__UIQtoMCtNGPndrSxnRMw%3A1704480735154&ei=30-YZYGJCd2B0PEPm82CyAU&oq=recreation+versus+re&gs_lp=Egxnd3Mtd2l6LXNlcnAiFHJlY3JlYXRpb24gdmVyc3VzIHJlKg0IABAAGIAEGIoFGJECMgsQABiABBiKBRiRAjIHEAAYgAQYCjIGEAAYFhgeSMUaUNEDWIkMcAF4AZABAJgBZaABtgaqAQM5LjG4AQHIAQD4AQHCAgoQABhHGNYEGLADwgINEAAYRxjWBBjJAxiwA8ICDhAAGIAEGIoFGJIDGLADwgINEAAYgAQYigUYQxiwA8ICEBAAGIAEGIoFGEMYsQMYyQPCAgsQABiABBiKBRiSA8ICDRAAGIAEGIoFGEMYsQPCAgoQABiABBiKBRhDwgIFEAAYgATCAg4QABiABBiKBRixAxiDAcICBRAuGIAEwgILEC4YgAQYxwEYrwHCAgoQABgWGB4YDxgKwgIIEAAYFhgeGAriAwQYACBBiAYBkAYK&sclient=gws-wiz-serp).